pkgs/by-name/kc/kittycad-kcl-lsp/package.nix

```
@@ -0,0 +1,31 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  pkg-config,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "kittycad-kcl-lsp";
+  version = "0.1.61";
+
+  src = fetchFromGitHub {
+    owner = "KittyCAD";
+    repo = "kcl-lsp";
+    rev = "v${version}";
+    hash = "sha256-VtrR4v0BJWYdoYFDJpWnmVqDhZMlPGm+g9yjxYfcFxQ=";
+  };
+
+  cargoHash = "sha256-51eFOJnc/GqgXtfVx/omR+KuC7x/oKGGR+s0z6nKXBg=";
+
+  nativeBuildInputs = [ pkg-config ];
+
+  meta = with lib; {
+    description = "KittyCAD Language Server. This is unrelated to https://www.kcl-lang.io/";
+    homepage = "https://github.com/KittyCAD/kcl-lsp";
+    license = licenses.mit;
+    maintainers = with maintainers; [ jljox ];
+    mainProgram = "kittycad-kcl-lsp";
+    meta.changelog = "https://github.com/KittyCAD/kcl-lsp/releases/tag/v${version}";
```

```suggestion
    changelog = "https://github.com/KittyCAD/kcl-lsp/releases/tag/v${version}";
```
`meta` is repeated.
Add `changelog` after `description`.